dev/tools/localization/gen_l10n.dart

```diff
@@ -744,6 +491,19 @@ class LocalizationsGenerator {
     outputFile = _fs.file(path.join(l10nDirectory.path, outputFileString));
   }
 
+  static bool _isValidClassName(String className) {
+    // Dart class name cannot contain non-alphanumeric symbols
+    if (className.contains(RegExp(r'[^a-zA-Z\d]')))
```

Comment:
Shouldn't this also allow underscores? [If you're not allowing underscores because it violates a style you want to enforce, then note that while Dart allows them, the style of the output doesn't.]

Reply:
Added an underscore to the REs